modules/migration-about-configuring-proxies.adoc

```diff
@@ -132,7 +132,26 @@ spec:
     type: Deny
 ----
 
-[id="configuring-supplemental-groups-for-rsync-pods{context}"]
+[id="choosing-alternate-endpoints-for-data-transfer_{context}"]
+=== Choosing alternate endpoints for data transfer
+
+By default, DVM uses OpenShift Route as an endpoint to transfer PV data to destination clusters. Endpoints are created on the destination cluster. Alternatively, if cluster topologies allow, you can choose other type of supported endpoints.
+
```

Comment:
Question: Does "Endpoints are created on the destination cluster" tell the user something they don't know from the first sentence?

Reply:
@RichardHoch i think that statement is not needed

Reply:
I agree. Removed.